### PR TITLE
Pot on prod build

### PR DESCRIPTION
**Pull request by ataylorme**
## Description

Addresses issue #314.

* Removes translation file generation from development builds
* Adds a new `export:generatePotFile` configuration variable to control `.pot` file generation on production builds
* Updates the Localization Component to only run `load_theme_textdomain` if `.pot` files exist
* Updates `README` documentation

## List of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] This pull request relates to a ticket.
- [x] This code is tested.
- [ ] This change has been added to CHANGELOG.md
- [x] I want my code added to WP Rig.

### CODE REVIEW

**Comment by ataylorme** [comment]
@felixarntz I would like to get your thoughts on the changes to the localization component.

File: inc/Localization/Component.php [hunk 1]
```diff
@@ -30,7 +61,10 @@ public function get_slug() : string {
 	 * Adds the action and filter hooks to integrate with WordPress.
 	 */
 	public function initialize() {
-		add_action( 'after_setup_theme', array( $this, 'action_load_textdomain' ), 1 );
+		// Only add textdomain actions if there are .pot files.
+		if ( $this->pot_files_exist() ) {
```
Comment (felixarntz) [inline inc/Localization/Component.php]: @ataylorme Just double-checking, is it really okay to run this logic on every page load?

Comment (ataylorme) [reply inc/Localization/Component.php]: @felixarntz if we don't do the check then `action_load_textdomain` could run without any valid files. Is that acceptable? If not, can you provide an alternative solution?

Comment (mor10) [reply inc/Localization/Component.php]: Might be a dumb idea, but would caching help here?

Comment (ataylorme) [reply inc/Localization/Component.php]: I think storing the result of `pot_files_exist` in a cache, like a transient, would be less efficient than just running the function

Comment (bamadesigner) [reply inc/Localization/Component.php]: WordPress Core has logic that checks if the file is readable/exists and, if not, doesn't load so we don't need this logic.

**Comment by bamadesigner** [comment]
Thanks @ataylorme!

**Pull request merged**